peerconnection.go

```diff
@@ -1856,8 +1856,9 @@ func (pc *PeerConnection) generateUnmatchedSDP(useIdentity bool) (*sdp.SessionDe
 		if len(audio) > 0 {
 			mediaSections = append(mediaSections, mediaSection{id: "audio", transceivers: audio})
 		}
-		for i := uint32(0); i < pc.sctpTransport.dataChannelsRequested; i++ {
```

Comment:
This was incorrect before. We only want one media section, this would be create one for every DataChannel.